### DISCUSSION TITLE
Can you please make a checker?

### DISCUSSION THREAD

**Discussion by ashwin3454** [qxxa/instahunt#2] [Ideas]
Can you please make a checker for usernames that was already taken or not? @qxxa 
like..
generate random usernames of [3 to 5] letters. and save to a txt file,
then check usernames from the list file,
checks: (already exist, not available, available)

**Comment by qxxa** [comment]
Hi @Ashwin3454,

Thanks for the idea. I had thought of adding a checker feature, but I’ve been busy. I’ll start working on it soon.

**Reply by ashwin3454** [reply]
thank you so much brother. I hope you can do it ASAP.

**Reply by qxxa** [reply]
Hey @Ashwin3454 , I've just added the username checker to my script. You can add a list of usernames to the usernames.txt file and proxies to the proxies.txt file and run the checker.
